vms/platformvm/block/builder/helpers_test.go

```diff
@@ -303,8 +292,16 @@ func defaultState(
 	return state
 }
 
-func defaultConfig(t *testing.T, f fork) *config.Config {
-	c := &config.Config{
+func defaultConfig(f upgradetest.Fork) *config.Config {
+	upgrades := upgradetest.GetConfigWithUpgradeTime(f, time.Time{})
+	// This package neglects fork ordering
+	upgradetest.SetTimesTo(
+		&upgrades,
+		min(f, upgradetest.ApricotPhase5),
```

Comment:
Why Apricot 5?

Reply:
This was done just to align with the previous code... This really should not do this (begs into question the tests that use this tbh). But without this hack - the tests fail. Hoping to keep this PR small.